src/code_8086A3C.c

```diff
@@ -20,6 +21,40 @@ extern void sub_80855E4(void *);
 extern void sub_8085860(s32 r0, u32 r1);
 extern void sub_8068FE0(struct DungeonEntity *, u32, u32);
 extern void sub_8097FA8(u32);
+extern void sub_80858AC(void *, u32);
+extern void sub_8052910(u8 *);
+extern void sub_806CDD4(struct DungeonEntity *, u32, u32);
+extern void sub_80869E4(struct DungeonEntity *, u32, u32, u32);
+extern void sub_8083E88(u32);
+extern void sub_803E708(u32, u32);
+extern void sub_8086448(void);
+extern void sub_80862BC(struct DungeonEntity *);
+extern void sub_8086E74(struct DungeonEntity *);
+extern void SpriteLookAroundEffect(struct DungeonEntity *);
+extern void sub_8086A54(struct DungeonEntity *);
+extern void PlaySoundEffect(u32);
+extern void sub_808563C(void *);
+
+// X / Y Coords??
+struct subStruct_Skarmony
```

Comment:
Should be "Skarmory". This is misspelled in a few places.

Reply:
Fixed!